packages/patternfly-4/react-charts/src/components/ChartDonutUtilization/ChartDonutThreshold.js

```diff
@@ -0,0 +1,258 @@
+  children,
+  data = [],
+  donutOrientation = 'left',
+  legendOrientation,
+  standalone = true,
+  themeColor,
+  themeVariant,
+  x,
+  y,
+
+  // destructure last
+  theme = getDonutThresholdStaticTheme(themeColor, themeVariant),
+  donutHeight = theme.pie.height,
+  donutWidth = theme.pie.width,
+  height = theme.pie.height,
+  innerRadius = ((donutHeight || donutWidth) - 34) / 2,
+  width = theme.pie.width,
+  ...rest
+}) => {
+  // Returns computed data representing pie chart slices
+  const getComputedData = () => {
+    const computedData = [];
+    let prevYVal = 0;
+    data.forEach(_data => {
+      const datum = [{ ..._data }];
+      const xVal = getDatumXVal(datum);
+      const yVal = getDatumYVal(datum);
+      computedData.push({ x: xVal, y: yVal ? Math.abs(yVal - prevYVal) : 0 });
+      prevYVal = yVal;
+    });
+    computedData.push({ y: prevYVal ? Math.abs(100 - prevYVal) : 0 });
+    return computedData;
+  };
+
+  // Returns a data point value via the x accessor
+  const getDatumXVal = datum => {
+    const result = map(datum, property(x || 'x'));
+    return result.length ? result[0] : undefined;
+  };
+
+  // Returns a data point value via the y accessor
+  const getDatumYVal = datum => {
+    const result = map(datum, property(y || 'y'));
+    return result.length ? result[0] : 0;
+  };
+
+  // Returns the horizontal shift for the dynamic utilization donut cart
+  const getDynamicDonutDx = (dynamicTheme, orientation) => {
+    switch (orientation) {
+      case 'left':
+        return Math.round((theme.pie.width - dynamicTheme.pie.width) / 2);
+      case 'right':
+        return -Math.round((theme.pie.width - dynamicTheme.pie.width) / 2);
+      default:
+        return 0;
+    }
+  };
+
+  // Returns the vertical shift for the dynamic utilization donut cart
+  const getDynamicDonutDy = dynamicTheme => Math.round((theme.pie.height - dynamicTheme.pie.height) / 2);
+
+  // Render dynamic utilization donut cart
+  const renderChildren = () =>
+    React.Children.toArray(children).map(child => {
+      const datum = [{ ...child.props.data }];
+      const yVal = getDatumYVal(datum);
+      const orientation = child.props.donutOrientation || donutOrientation;
+      const dynamicTheme =
+        child.props.theme ||
+        getDonutThresholdDynamicTheme(child.props.themeColor || themeColor, child.props.themeVariant || themeVariant);
+      return React.cloneElement(child, {
+        donutDx: child.props.donutDx || getDynamicDonutDx(dynamicTheme, orientation),
+        donutDy: child.props.donutDy || getDynamicDonutDy(dynamicTheme),
+        donutHeight: child.props.donutHeight || dynamicTheme.pie.height,
+        donutOrientation: orientation,
+        donutWidth: child.props.donutWidth || dynamicTheme.pie.width,
+        endAngle: child.props.endAngle || 360 * (yVal ? yVal / 100 : 100),
+        height: child.props.height || height,
+        showStatic: child.props.showStatic !== undefined || false,
```

Comment:
`child.props.showStatic !== undefined` will evaluate to `true` even if `child.props.showStatic === false`. This should probably just be `showStatic: child.props.showStatic`.  If `child.props.showStatic` is undefined, it will evaluate as false if used in a boolean expression, so I think it's okay to let the undefined prop fall through.

Reply:
Also, the `|| false` is not necessary, since `booleanValue || false` will evaluate to the same value as `booleanValue`

Reply:
The showStatic prop isn't necessary here since the underlying component already has a default. I'll just remove it.